classes/Http/Form/SignupForm.php

```diff
@@ -56,8 +51,6 @@ public function validateAll($action = 'create')
         $validEmail        = $this->validateEmail();
         $validFirstName    = $this->validateFirstName();
         $validLastName     = $this->validateLastName();
-        $validCompany      = $this->validateCompany();
-        $validTwitter      = $this->validateTwitter();
```

Comment:
Both of these methods return `true`, I think it's better to remove them rather than suggesting that some kind of validation actually occurs.